src/Donation/TransactionCallbackHandler.php

```diff
@@ -15,15 +16,17 @@ class TransactionCallbackHandler
 {
     private $router;
     private $entityManager;
-    private $mailer;
+    private $mailerTransactional;
     private $donationRequestUtils;
+    private $transactionRepository;
 
-    public function __construct(UrlGeneratorInterface $router, ObjectManager $entityManager, MailerService $mailer, DonationRequestUtils $donationRequestUtils)
+    public function __construct(UrlGeneratorInterface $router, ObjectManager $entityManager, MailerService $mailerTransactional, DonationRequestUtils $donationRequestUtils, TransactionRepository $transactionRepository)
     {
         $this->router = $router;
         $this->entityManager = $entityManager;
-        $this->mailer = $mailer;
+        $this->mailerTransactional = $mailerTransactional;
```

Comment:
agree with @B-Galati, this change is useless

Reply:
@cacahouwete you forgot this one